### DISCUSSION TITLE
[Build file size too big]

### DISCUSSION THREAD

**Discussion by iamleson98** [tailwindlabs/tailwindcss#2009] [Help] [answered]
**hello guys**

I'm new to tailwindcss and very interested in learning it. I followed this post https://www.smashingmagazine.com/2020/02/tailwindcss-react-project/ to add tailwindcss to my existing reactjs project
After i run the command **npm run build**, this is the css files in the build directory(**only tailwind generated code, not my custom css**)
![image](https://user-images.githubusercontent.com/28799254/86930452-3d245080-c161-11ea-9f07-dad00c364fd4.png)

Could anybody please tell me why the css file is too big and Does it affect page load speed ?
How to reduce the file size ?
Thank you!

**Comment by adamwathan** [comment, marked as answer]
Hey! Check out this page in our docs on building for production and optimizing file size:

https://tailwindcss.com/docs/controlling-file-size

Should help a lot 👍

**Reply by iamleson98** [reply]
Hi mr @adamwathan.
![Screenshot (46)_LI](https://user-images.githubusercontent.com/28799254/87216409-a8407380-c369-11ea-8fe1-2079b14517d7.jpg)

I insert different class texts to an element, but visual studio code show me this warning.
What should i do ?

Thank you!

**Reply by Apostolique** [reply]
You could add something like this to your tailwind config:
```js
transitionProperty: {
    'transform-opacity': 'transform, opacity',
},
```
Then you can use it with: `transition-transform-opacity`.
